specification/network/resource-manager/Microsoft.Network/2017-08-01/checkDnsAvailability.json

```diff
@@ -51,7 +51,7 @@
           {
             "name": "domainNameLabel",
             "in": "query",
-            "required": false,
```

Comment:
Was this a mistake previously? 

If they didn't pass it in would this fail?

This shouldn't be a binary breaking change, but it is technically be a source-level-breaking change (since someone could have omitted the parameter and now would have to pass one).

Reply:
domainNameLabel is a required property on the service. If users were not passing the parameter, the service would throw an exception. 

Yes, it is a breaking change on the client but it is required and i dont expect any customers to get impacted since we were anyway throwing .

This is a bug that we are fixing on the client

Reply:
I figured as much -- it's not likely to cause a breaking change in the generated code unless someone used the function without the parameter, which would have failed... works for me.